File: python/37 ArmstrongNumber.py
def isArm(num):
    org=num
    temp=num
    digit=0
    while temp>0:
        digit+=1
        temp//=10
    res=0
    while num>0 :
        res+=(num%10)**digit
        num//=10
    if org==res:
        print("Yes")
    else:
        print("No")
num=int(input("Enter a number: "))
isArm(num)

File: java/37 ArmstrongNumber.java
package code;
import java.util.Scanner;
public class Test {
	static void isArm(int num) {
		int org=num;
		int temp=num;
		int digit=0;
		while(temp>0) {
			digit++;
			temp/=10;
		}
		double res=0;
		while(num>0) {
			res+=Math.pow((num%10),digit);
			num/=10;
		}
		if(org==res) {
			System.out.println("Yes");
		}
		else {
			System.out.println("No");
		}
	}
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter a nummber: ");
		int num1=sc.nextInt();
		isArm(num1);
	}
}
